cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -968,7 +972,7 @@ public void getWorkflowSchedules(HttpRequest request, HttpResponder responder,
         specList.add(entry.getValue());
       }
     }
-    responder.sendJson(HttpResponseStatus.OK, specList);
+    responder.sendJson(HttpResponseStatus.OK, GSON.toJsonTree(specList));
```

Comment:
Do we need that change? Won't responder.sendJson() use the right codec by itself?